libpod/events.go

```diff
@@ -38,6 +38,24 @@ func (c *Container) newContainerEvent(status events.Status) {
 		Attributes: c.Labels(),
 	}
 
+	if status == events.Create && c.runtime.config.Engine.EventsContainerCreateInspectData {
+		err := func() error {
+			data, err := c.inspectLocked(true)
+			if err != nil {
+				return err
+			}
+			rawData, err := json.Marshal(data)
+			if err != nil {
+				return err
+			}
+			e.Details.ContainerCreateInspectData = string(rawData)
+			return nil
+		}()
+		if err != nil {
+			logrus.Errorf("Unable to add inspect data to container-create event: %v", err)
```

Comment:
Seems like a customer who wants full audit data would prefer that container creation actually fail if we can't provide this?

Reply:
I was contemplating it but couldn't decide.  But you changed my mind.  I also believe it's valid to fail in this case.

Reply:
Done.  Note that I was very careful not to change other callers to write events.  Almost all are using `defer` and changing that would be a (too) big PR.

I also took the occasion to rename PODMAN_CONTAINER_CREATE_INSPECT_DATA to PODMAN_CONTAINER_INSPECT_DATA ... just in case we want more container events to contain this data in the future.